src/identities/CommandClaim.ts

```diff
@@ -1,5 +1,6 @@
 import type PolykeyClient from 'polykey/dist/PolykeyClient';
 import type { WebSocketClient } from '@matrixai/ws';
+import type { GestaltId } from 'polykey/dist/gestalts/types';
```

Comment:
You can import types from the `polykey` directly now. Us 
```suggestion
import type { GestaltId } from 'polykey';
```